web-ui/src/main/java/org/vaadin/example/queryview/PauseButton.java

```diff
@@ -0,0 +1,27 @@
+package org.vaadin.example.queryview;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+
+class PauseButton extends Button {
```

Comment:
I am not entirely sure why we need the ability to pause a triplet

Reply:
Pausing is the light version of remove.
Dont know if you get less results because of this row? Pause it rerun. If not keep it otherwise remove it.
Just thought it would be a nice to have. Can be easily removed. Lets discuss that on Thursday.

Reply:
Interesting idea! But we should make the functionality clearer maybe by greying out the row when it is paused.